### ISSUE TITLE
[BUG] ssh/scp completion duplicate all items in each groups: host/domain/ip address

### ISSUE THREAD

**Issue by fluxtendu** [Aloxaf/fzf-tab#115]
Hi,

fzf-tab seems to not differentiate "host", "domain" and "ip address" groups. So all hosts, domain and ip are all in each group host/domain/ip address.
If I `toggle-fzf-tab` , no duplicate and ip address and host are correctly differentiated (no domain group)

**Steps to reproduce the behavior:**
1. Add some items in /etc/hosts and ~/.ssh/config
2. Type 'SSH '
3. Press <kbd>Tab</kbd>

**Screenshots**
with fzf-tab:
<img src="https://user-images.githubusercontent.com/12612144/91643729-17ffe000-ea36-11ea-8f98-e1ee86ba40b8.png" width="314" height="507">
without fzf-tab:
![fzf-tab-disabled+SSH+TAB-](https://user-images.githubusercontent.com/12612144/91643739-336aeb00-ea36-11ea-9021-9c2ea3d2e7e3.png)

**Environment (please complete the following information):**
 - OS: Raspbian 10 (buster)
 - zsh 5.7.1
 - fzf-tab https://github.com/Aloxaf/fzf-tab/commit/4108af7dc37fc89f59ada74eab79b2037db311fc

**Maybe related parts of zshrc**
```
# prezto
if [[ -s "${ZDOTDIR:-$HOME}/.zprezto/init.zsh" ]]; then
  source "${ZDOTDIR:-$HOME}/.zprezto/init.zsh"
fi
# fzf
source /usr/share/doc/fzf/examples/key-bindings.zsh
source /usr/share/zsh/vendor-completions/_fzf
export FZF_DEFAULT_OPTS='
--color dark,hl:2,hl+:148,fg:14,bg+:0,fg+:15
--color info:6,prompt:4,spinner:4,pointer:3,marker:3
--ansi
'
export FZF_DEFAULT_COMMAND="fdfind --type f --exclude .git --hidden --no-ignore --follow --color=always"
export FZF_CTRL_T_COMMAND="fdfind --exclude .git --hidden --no-ignore --follow --color=always"
export FZF_ALT_C_COMMAND="fdfind --type d --hidden --no-ignore --follow --color=always"
# fzf-tab
zstyle ':completion:*' format '[%d]'
setopt glob_dots
```
**Maybe related parts of zpreztorc**
```
zstyle ':prezto:load' pmodule \
  'environment' \
  'terminal' \
  'editor' \
  'history' \
  'directory' \
  'spectrum' \
  'utility' \
  'completion' \
  'fzf-tab' \
  'git' \
  'syntax-highlighting' \
  'history-substring-search' \
  'autosuggestions' \
  'prompt'
```

**Comment by Aloxaf** [comment]
The code causes this bug is here: https://github.com/sorin-ionescu/prezto/blob/ff91c8d410df3e6141248474389051c7ddcaf80a/modules/completion/init.zsh#L159-L164

It seems that `ignored-patterns` is ignored when enable fzf-tab.

**Comment by fluxtendu** [comment]
I have added to my .zshrc :
`ip6-allnodes` and `ip6-allrouters` to hosts-host ignored-patterns
`ff02::1`, `ff02::2` and `127.0.1.<->` to hosts-ipaddr ignored-patterns
with:
```
zstyle ':completion:*:(ssh|scp|rsync):*:hosts-host' ignored-patterns '*(.|:)*' loopback ip6-loopback localhost ip6-localhost broadcasthost ip6-allnodes ip6-allrouters
zstyle ':completion:*:(ssh|scp|rsync):*:hosts-ipaddr' ignored-patterns '^(<->.<->.<->.<->|(|::)([[:xdigit:].]##:(#c,2))##(|%*))' '127.0.0.<->' '255.255.255.255' '::1' 'fe80::*' 'ff02::1' 'ff02::2' '127.0.1.<->'
```
and they are effectively ignored... So the ignored-patterns is not entirely ignored, maybe it's the pattern at the beginning?!

(I have also tried to add all the SSH/SCP/RSYNC lines but without improvements.)

**Comment by Aloxaf** [comment]
Closed by 7f8254a184d159016587f586c13f6686c17f3e7a
